src/DSharp.Compiler.Tests/DSharp.Compiler.Tests.csproj

```diff
@@ -12,6 +12,783 @@
     <None Remove="Source\Validation\AllowSystemNamespace\**" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Can we remove this item group as it should collected with the item group at the bottom. If something is missing can you update that group?